packages/medusa/src/services/order-edit.ts

```diff
@@ -537,6 +542,60 @@ export default class OrderEditService extends TransactionBaseService {
     await lineItemAdjustmentServiceTx.createAdjustments(localCart)
   }
 
+  /**
+   * Calculate totals for order edit items and decorate the items.
+   *
+   * @param orderEdit
+   */
+  async decorateItemsTotals(orderEdit: OrderEdit) {
+    const manager = this.transactionManager_ ?? this.manager_
+    const totalsServiceTx = this.totalsService_.withTransaction(manager)
+    const newTotalsServiceTx = this.newTotalsService_.withTransaction(manager)
+
+    const { order_id, items } = await this.retrieve(orderEdit.id, {
+      select: ["id", "order_id", "items"],
+      relations: ["items", "items.tax_lines", "items.adjustments"],
+    })
+
+    const order = await this.orderService_
+      .withTransaction(manager)
+      .retrieve(order_id, {
+        relations: [
+          "discounts",
+          "discounts.rule",
+          "gift_cards",
+          "region",
+          "items",
+          "items.tax_lines",
```

Comment:
thought: could we explicitly add `items.variant` even though it is eager at the moment. (you are not obliged but it does not change that much here but it is very helpful for the typeorm pr :))

Reply:
will add :)